<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="jayadeeps-MacBook-Pro.local" failures="1" tests="2" name="Default test(failed)" time="0.047" errors="0" timestamp="2021-06-15T17:55:19 IST">
  <testcase classname="DemoTestNG.RetryFailedTestNG" name="test3" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at DemoTestNG.RetryFailedTestNG.test3(RetryFailedTestNG.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="DemoTestNG.RetryFailedTestNG" name="test2" time="0.002"/>
  <testcase classname="DemoTestNG.RetryFailedTestNG" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
